lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -435,6 +435,248 @@
     }
   }
 
+  .batch-enrollment, .batch-beta-testers {
+    textarea {
```

Comment:
In general we avoid using element based selectors, which there are a number of in these SASS changes. Please use class names here that make sense for the controls and remove all element selectors (textarea, input, h3, ul, etc)